src/confirm-submission.html

```diff
@@ -32,8 +77,8 @@ <h1 class="margin-bottom-5">Step 4 of 4: Confirm submission</h1>
               </p>
             </section>
 
-            <a href="{{ '/instructor-details' | url }}" class="usa-button usa-button--outline margin-y-4" type="submit">Go back to instructor details</a>
-            <a href="{{ '/' | url }}" class="usa-button margin-y-4" type="submit">Submit</a>
+            <button onclick="location.href = '/instructor-details'" class="usa-button usa-button--outline margin-y-4" type="submit">Go back to instructor details</button>
```

Comment:
@mahoneycm this fails in preview, but you could use something like `window.location.host` to ensure the right base URL gets used in all environments.

**Steps to reproduce**
1. Go to [homepage](https://federalist-d5e7c07c-6ffa-4b8e-935f-49a7df24a505.sites.pages.cloud.gov/preview/uswds/uswds-sandbox/cm-zombie-defense-update-buttons/).
2. Click `Sign in` button.
3. Cloud pages 404 appears.